core/src/main/kotlin/com/alecstrong/sql/psi/core/psi/mixins/SelectStmtMixin.kt

```diff
@@ -46,6 +50,25 @@ internal abstract class SelectStmtMixin(
   override fun annotate(annotationHolder: SqlAnnotationHolder) {
     super.annotate(annotationHolder)
 
+    val invalidBinaryEqualityBindExpr = exprList.find { child ->
```

Comment:
its still valid to do `WHERE ? = some_column` so I wouldn't want to fail here

Reply:
I was thinking about that. What if the failure only occurs if the RHS is also a `SqlBindExpr`?

Reply:
I also updated to test for all binary expressions

Reply:
yep im cool with LHS and RHS cant both be a bind expr. I'd recommend just doing that in a new mixin for binary expressions then instead of having it live in SelectStmtMixin

Reply:
I can add a mixin to all the binary comparison expressions, but is there a way to add something that includes all the different types of expressions in a synthetic one?

Something like:

```
binary_comparison_expr ::= [ binary_boolean_expr | binary_equality_expr | binary_like_expr | is_expr | between_expr | in_expr ]
```

Reply:
that would need a ton of changes downstream so probably not ideal. you could have a single mixin kotlin class like

```
abstract class BinaryMixin : SqlCompositeElement(node) {
  abstract fun getExprList(): List<SqlExpr>

  fun annotate() {
    ...
  }
}
```

and then you just have to repeat yourself in the bnf but i think thats fine

Reply:
Just pushed that update